cmd/run.go

```diff
@@ -107,34 +112,37 @@ dapr run --app-id myapp --app-port 3000 --app-protocol grpc -- go run main.go
 			}
 		}
 
-		output, err := standalone.Run(&standalone.RunConfig{
-			AppID:              appID,
-			AppPort:            appPort,
-			HTTPPort:           port,
-			GRPCPort:           grpcPort,
+		sharedRunConfig := &standalone.SharedRunConfig{
```

Comment:
since RunConfig composes the SharedRunConfig struct, you can created one RunConfig struct here with all fields instead of this way.

Reply:
To initialise we will have to do it separately only as far as I can see here - https://gobyexample.com/struct-embedding

While accessing it possible to access only with container struct until both does not have same fields.